### DISCUSSION TITLE
Compartments

### DISCUSSION THREAD

**Discussion by hariszaf** [GeomScale/dingo#55] [Ideas]
To model a community, we will need to track down the reactions and their corresponding metabolites, that 
take place in the extracellular space. 

It is my belief that it would benefit our efforts the most, to use models in the Systems Biology Markup Language (SBML)
format to this end. 

This means both to implement a function for reading such files (the [libSBML](http://sbml.org/Software/libSBML/Downloading_libSBML) library will probably be of help for this, but also 
to add an attribute in our model, e.g. `model.extracellular` to keep track of these metabolites. 

@TolisChal @vissarion  what 's your thoughts?

**Comment by vissarion** [comment]
Adding a function reading such files sounds reasonable. Regarding the attribute another choice would be to create a new class `community_model` that inherits model and there add the `extracellular` attribute.

**Comment by Kundanagrawalofficial** [comment]
The Systems Biology Markup Language (SBML) is a file format for representing computational models in a declarative form that can be exchanged ...
